### PR TITLE
assignment2b

### PR DESCRIPTION
**Pull request by alejandrofdezmerino**

### CODE REVIEW

File: Assignment2/alejandrofdezmerino-fromRDFtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+"@context": {
+"ex": "http://www.example.com/",
+"rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+  "hasName": "ex:name",
+  "hasTemperature": "ex:int",
+  "atTime": "rdfs:date"
+},
```
Comment (paoespinozarias) [inline Assignment2/alejandrofdezmerino-fromRDFtoJSON-LD.jsonld:8]: "includes", "hasOwner", "hasMeasurement" are missing.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

**Comment by alejandrofdezmerino** [comment]
New update assignment 2b

File: Assignment2/alejandrofdezmerino-fromJSONtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	"name": "vocab:name",
+	"hometown": "vocab:hometown",
+	"age": "vocab:age",
+	"almaMater": "vocab:almaMater",
+	"parents": "vocab:parents",
+	"subjects": "vocab:subjects",
+	"teachers": "vocab:teachers",
+	"description": "vocab:description"
+},
+	"name": "Alejandro Fernandez Merino",
```
[the remark this answers is not in the document]
Comment (paoespinozarias) [reply Assignment2/alejandrofdezmerino-fromJSONtoJSON-LD.jsonld:14]: Still pending to add "@type"

File: Assignment2/alejandrofdezmerino-fromJSONtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	"teachers": "vocab:teachers",
+	"description": "vocab:description"
+},
+	"@id": "http://www.example.com/ontology/myself",  
+	"name": "Alejandro Fernandez Merino",
+	"hometown": "Madrid",
+	"age": "23",
+	"almaMater": "Universidad Politecnica de Madrid",
+	"parents": "Africa y Julian",
+	"subjects":	{
```
Comment (paoespinozarias) [inline Assignment2/alejandrofdezmerino-fromJSONtoJSON-LD.jsonld:19]: Define it as an array [ ]
Then you don't need to define "subject1": , "subject2": ,etc. Delete these names and leave only the name, teachers, description values inside curly braces.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

**Comment by alejandrofdezmerino** [comment]
New update assignment 2b

**Review by paoespinozarias — APPROVED** [review]

**Pull request merged**